src/contexts/Dashboard.js

```diff
@@ -138,6 +140,17 @@ function reducer(state, { type, payload }) {
       };
     }
 
+    case UPDATE_UNCOLLECTED_FEES_DATA: {
+      const { uncollectedFees } = payload;
+      return {
+        ...state,
+        feesData: {
+          ...state.feesData,
```

Comment:
```suggestion
      
```
Are you sure you need old data?. Anyway the new objects will override the old ones.